mapstruct-lombok/build.gradle

```diff
@@ -1,6 +1,5 @@
 plugins {
     id 'java'
-    id 'net.ltgt.apt' version '0.9'
```

Comment:
I just found  that this is not `apt` problem, and `gradle test` task is work well.But the exception will happen when I click run button in IntelliJ IDEA. Meanwhile as you can see in below image, the generate folder is empty.

![1](https://user-images.githubusercontent.com/3816374/29112888-e751a706-7d21-11e7-8883-048248d95b16.png)